### PR TITLE
add help func

### PR DESCRIPTION
**Pull request by runzexia**

### CODE REVIEW

File: pkg/manager/runtime_env/help.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+package runtime_env
+
+import (
+	"fmt"
+	"strings"
+
+	"openpitrix.io/openpitrix/pkg/models"
+)
+
+func LabelStringToMap(labelString string) (map[string]string, error) {
```
Comment (zheng1) [inline pkg/manager/runtime_env/help.go]: use `url.ParseQuery`

File: pkg/manager/runtime_env/help.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+			return nil, fmt.Errorf("bad label format %v", labelString)
+		}
+		mapLabel[labelArray[0]] = labelArray[1]
+	}
+
+	return mapLabel, nil
+}
+
+
+func SelectorStringToMap(selectorString string) (map[string][]string, error) {
```
Comment (zheng1) [inline pkg/manager/runtime_env/help.go]: use `url.ParseQuery`

File: pkg/manager/runtime_env/help.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+			return nil, fmt.Errorf("bad selector format %v", selectorMap)
+		}
+		labelArray[0] = strings.TrimSpace(labelArray[0])
+		labelArray[1] = strings.TrimSpace(labelArray[1])
+		selectorMap[labelArray[0]] = append(selectorMap[labelArray[0]], labelArray[1])
+	}
+	return selectorMap, nil
+}
+
+func LabelMapDiff(oldLabelMap, newLabelMap map[string]string) (additions, deletions map[string]string) {
```
Comment (zheng1) [inline pkg/manager/runtime_env/help.go]: This function is no need to add until we communicate with front-end.

File: pkg/manager/runtime_env/help.go [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+			}
+		}
+		if i == 0 {
+			oldLabelMap, newLabelMap = newLabelMap, oldLabelMap
+		}
+	}
+	return additions, deletions
+}
+
+func LabelStructToMap(labelStructs []*models.RuntimeEnvLabel) map[string]string {
```
Comment (zheng1) [inline pkg/manager/runtime_env/help.go]: We return label with `pb.RuntimeEnvLabel`, this func is no need for the moment.

**Comment by runzexia** [comment]
Done.
To avoid unpredictable errors caused by `QueryUnescape`, the parse method is rewritten without directly using `url.ParseQuery()`

File: pkg/manager/runtime_env/help.go [hunk 5]
```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
```
Comment (zheng1) [inline pkg/manager/runtime_env/help.go]: I think there is no problem. Please change the file name to `helper.go`, and then rebase your commits.

**Pull request merged**